asm/custom-functions/src/game/flag_managers.rs

```diff
@@ -131,3 +131,220 @@ impl DungeonflagManager {
         unsafe { (*Self::get_global(scn_idx))[1] & 0xF }
     }
 }
+
+// Flag enums
+#[repr(u16)]
+#[derive(Copy, Clone, Hash, PartialEq, Eq)]
+#[allow(non_camel_case_types)]
+pub enum ITEMFLAGS {
```

Comment:
```suggestion
pub enum Itemflag {
```
I would follow the rust naming convention here and use singular